### ISSUE TITLE
$("#loading").show(); issue

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#872]
Originally reported on Google Code with ID 602

```
Hey there,

I have an issue where my "#loading" div is being shown the entire time, and is not
hiding after the JSON call is being made.

Here's what my javascript looks like:

loading: function(bool) {
if (bool)  $('#loading').show(); 
else  $('#loading').hide(); 
           }

And the DIV for the loading is as such:

<div id="loading" style="display:none;>
loading...
</div>

Basically, I load the page, and the "loading" div is shown, but it never hides.  It
is always static.

Anyone else run into this issue at all?  Any ideas?

Thanks.
```

Reported by `KyleRenke` on 2010-08-12 06:15:57
